<?xml version="1.0" ?>
<testsuites disabled="0" errors="1" failures="0" tests="2" time="34.020740032196045">
	<testsuite disabled="0" errors="1" failures="0" name="meraki_alert-d34p3e12" skipped="0" tests="2" time="34.020740032196045">
		<testcase classname="/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_alert-k0yj4why-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_alert/tasks/main.yml:7" name="[localhost] meraki: meraki_alert : Create network with type wireless auth_key={{ auth_key }}, state=present, org_name={{test_org_name}}, net_name=IntTestNetworkWireless, type=wireless, timezone=America/Chicago" time="2.720896">
			<system-out>{
&quot;changed&quot;: false,
&quot;data&quot;: {
&quot;enrollment_string&quot;: null,
&quot;id&quot;: &quot;N_624874448297687307&quot;,
&quot;name&quot;: &quot;IntTestNetworkWireless&quot;,
&quot;notes&quot;: null,
&quot;organization_id&quot;: &quot;133277&quot;,
&quot;product_types&quot;: [
&quot;wireless&quot;
],
&quot;tags&quot;: [],
&quot;time_zone&quot;: &quot;America/Chicago&quot;,
&quot;url&quot;: &quot;https://n110.meraki.com/IntTestNetworkWi/n/liMxucUb/manage/usage/list&quot;
},
&quot;deprecations&quot;: [
{
&quot;msg&quot;: &quot;Distribution Ubuntu 18.04 on host localhost should use /usr/bin/python3, but is using /usr/bin/python for backward compatibility with prior Ansible releases. A future Ansible release will default to using the discovered platform python for this host. See https://docs.ansible.com/ansible/2.10/reference_appendices/interpreter_discovery.html for more information&quot;,
&quot;version&quot;: &quot;2.12&quot;
}
],
&quot;response&quot;: &quot;OK (unknown bytes)&quot;,
&quot;status&quot;: 200
}</system-out>
		</testcase>
		<testcase classname="/root/ansible_collections/cisco/meraki/tests/output/.tmp/integration/meraki_alert-k0yj4why-ÅÑŚÌβŁÈ/tests/integration/targets/meraki_alert/tasks/main.yml:17" name="[localhost] meraki: meraki_alert : Update settings with check mode auth_key={{ auth_key }}, org_name={{test_org_name}}, net_name=IntTestNetworkWireless, state=present, default_destinations={'emails': ['{{ email_prefix }}@{{ email_domain }}', '{{ email_prefix }}2@{{ email_domain }}'], 'all_admins': False, 'snmp': False}, alerts=[{'type': 'gatewayDown', 'enabled': True, 'filters': {'timeout': 60}, 'alert_destinations': {'emails': ['{{ email_prefix }}@{{ email_domain }}', '{{ email_prefix }}2@{{ email_domain }}'], 'all_admins': False, 'snmp': False}}, {'type': 'usageAlert', 'enabled': True, 'filters': {'period': 1200, 'threshold': 104857600}, 'alert_destinations': {'emails': ['{{ email_prefix }}@{{ email_domain }}', '{{ email_prefix }}2@{{ email_domain }}'], 'all_admins': False, 'snmp': False}}]" time="31.299844">
			<error message="AttributeError: 'NoneType' object has no attribute 'read'" type="error">Traceback (most recent call last):
  File &quot;/root/.ansible/tmp/ansible-tmp-1602871004.1972342-143-15884166463361/AnsiballZ_meraki_alert.py&quot;, line 143, in &lt;module&gt;
    _ansiballz_main()
  File &quot;/root/.ansible/tmp/ansible-tmp-1602871004.1972342-143-15884166463361/AnsiballZ_meraki_alert.py&quot;, line 135, in _ansiballz_main
    invoke_module(zipped_mod, temp_path, ANSIBALLZ_PARAMS)
  File &quot;/root/.ansible/tmp/ansible-tmp-1602871004.1972342-143-15884166463361/AnsiballZ_meraki_alert.py&quot;, line 81, in invoke_module
    runpy.run_module(mod_name='ansible_collections.cisco.meraki.plugins.modules.meraki_alert', init_globals=None, run_name='__main__', alter_sys=True)
  File &quot;/usr/lib/python3.6/runpy.py&quot;, line 205, in run_module
    return _run_module_code(code, init_globals, run_name, mod_spec)
  File &quot;/usr/lib/python3.6/runpy.py&quot;, line 96, in _run_module_code
    mod_name, mod_spec, pkg_name, script_name)
  File &quot;/usr/lib/python3.6/runpy.py&quot;, line 85, in _run_code
    exec(code, run_globals)
  File &quot;/tmp/ansible_meraki_alert_payload_7t2_bwqa/ansible_meraki_alert_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_alert.py&quot;, line 333, in &lt;module&gt;
  File &quot;/tmp/ansible_meraki_alert_payload_7t2_bwqa/ansible_meraki_alert_payload.zip/ansible_collections/cisco/meraki/plugins/modules/meraki_alert.py&quot;, line 295, in main
  File &quot;/tmp/ansible_meraki_alert_payload_7t2_bwqa/ansible_meraki_alert_payload.zip/ansible_collections/cisco/meraki/plugins/module_utils/network/meraki/meraki.py&quot;, line 261, in get_nets
  File &quot;/tmp/ansible_meraki_alert_payload_7t2_bwqa/ansible_meraki_alert_payload.zip/ansible_collections/cisco/meraki/plugins/module_utils/network/meraki/meraki.py&quot;, line 447, in request
AttributeError: 'NoneType' object has no attribute 'read'
</error>
		</testcase>
	</testsuite>
</testsuites>
